src/panels/config/energy/dialogs/dialog-energy-gas-settings.ts

```diff
@@ -141,14 +160,13 @@ export class DialogEnergyGasSettings
         </ha-formfield>
         <ha-formfield
           .label=${this.hass.localize(
-            `ui.panel.config.energy.gas.dialog.cost_stat`
+            "ui.panel.config.energy.gas.dialog.cost_stat"
           )}
         >
           <ha-radio
             value="statistic"
             name="costs"
             .checked=${this._costs === "statistic"}
-            .disabled=${externalSource}
```

Comment:
This was wrong: A sensor or external statistics which tracks the total cost is supported both when the consumed gas is tracked by an external statistics or by a sensor. A price entity or a fixed price is only supported when the consumed gas is tracked by a sensor.

Same change for for grid and water.

Reply:
Hi @emontnemery,

I'm trying to understand why I cannot use a fixed price on external statistics but I cannot find any related documentation.

Could you please tell me where I can find more information about it? I doesn't make sense to me

Thanks!